utils/build-script

```diff
@@ -1178,14 +1221,38 @@ details of the setups of other systems or automated environments.""")
     if not host_cmake:
         print_with_argv0("Can't find CMake. Please install CMake.")
         return 1
+
+    # distcc usage.
+    host_distcc = None
+    host_distcc_pump = None
```

Comment:
I feel like these variables should go into the `Toolchain` object, `host_clang`.